### PR TITLE
Nip 336

### PR DESCRIPTION
**Pull request by khajan-ari**
created ansible scripts for dbbackup installation.
new tags added in site.yml
created two new groups for slave db.
added yml and template files for report and motech slave db.

### CODE REVIEW

File: site.yml [hunk 1]
```diff
@@ -42,3 +42,22 @@
...
+  hosts: slavedb_motech
+  remote_user: vagrant
+  serial: 1
+  tasks:
+   - include:  roles/db/tasks/dbbackup.yml
+  tags:
+   - dbbackupscript
+- name: configure and deploy dbbackupScripts in db server
+  hosts: slavedb_report
+  remote_user: vagrant
```
Comment (manjotnms) [inline site.yml]: Remote user should be grameen

File: files/dbbackup/bin/mysql-incremental [hunk 2]
```diff
@@ -0,0 +1,667 @@
+#!/bin/bash
+set -x
+#MySQL credentials to do backup
+MySQLUserName="root"
```
Comment (manjotnms) [inline files/dbbackup/bin/mysql-incremental]: Remove sample mysql credentials

File: files/dbbackup/conf/automysqlbackup.conf [hunk 3]
```diff
@@ -0,0 +1,270 @@
...
+# /etc/automysqlbackup/automysqlbackup.conf will overwrite them. The
+# declarations in here will supersede all other.
+
+# Edit $PATH if mysql and mysqldump are not located in /usr/local/bin:/usr/bin:/bin:/usr/local/mysql/bin
+#PATH=${PATH}:FULL_PATH_TO_YOUR_DIR_CONTAINING_MYSQL:FULL_PATH_TO_YOUR_DIR_CONTAINING_MYSQLDUMP
+
+# Basic Settings
+
+# Username to access the MySQL server e.g. dbuser
+CONFIG_mysql_dump_username='root'
```
Comment (manjotnms) [inline files/dbbackup/conf/automysqlbackup.conf]: Remove the mysql username and password

File: hosts [hunk 4]
```diff
@@ -26,6 +26,11 @@
 [health-monitor]
 192.168.200.4:1027 ansible_connection=local
 
+[slavedb_motech]
+192.168.200.6:1027
+[slavedb_report]
+192.168.200.66:1027
```
Comment (manjotnms) [inline hosts]: Reporting Server's configured Slave is 192.168.200.68

File: roles/db/tasks/dbbackup_report.yml [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+    owner=grameen
+    group=grameen
+  become: yes
+  become_method: sudo
+
+- name: Schedule cront for Full Db backup of Report
+  cron: >
+    name=ReprotDbFullBackup
+    minute="30"
+    hour="4"
```
Comment (manjotnms) [inline roles/db/tasks/dbbackup_report.yml]: Reporting DB backup time should modified as per reporting DB requirement

File: roles/db/templates/dbbackup/automysqlbackup.conf.j2 [hunk 6]
```diff
@@ -0,0 +1,274 @@
...
+CONFIG_rotation_weekly=7
+
+# Set rotation for monthly backups. VALUE*24hours
+CONFIG_rotation_monthly=150
+
+
+# Server Connection Settings
+
+# Set the port for the mysql connection
+#CONFIG_mysql_dump_port=3306
```
Comment (manjotnms) [inline roles/db/templates/dbbackup/automysqlbackup.conf.j2:104]: Move mysql port to variable

Comment (DeepakTiwariDeepak) [inline roles/db/templates/dbbackup/automysqlbackup.conf.j2:104]: its commented. so i think no need to remove.

**Pull request merged**